test/lib/datadog-transport-test.js

```diff
@@ -16,6 +16,8 @@ const should = require("chai").should();
 const DatadogTransport = require("../../lib/datadog-transport");
 
 describe("Datadog transport", () => {
+  afterEachScenario(nock.cleanAll);
```

Comment:
Maybe add a test where the circuit breaker is thrown (if possible)?

Reply:
Will fix once initial panic is over. 8-)